src/main/cpp/fielddata/FieldElementCalculator.h

```diff
@@ -76,21 +76,21 @@ class FieldElementCalculator
     frc::Transform3d m_calcCoralLeftAlliance = frc::Transform3d(
         frc::Translation3d(
             units::length::inch_t(0.0),
```

Comment:
I think you are colliding with Charlies PR

Reply:
When I merge his stuff in, I will fix it